test/verify/check-system-info

```diff
@@ -421,6 +425,65 @@ class TestSystemInfo(MachineCase):
 
         self.machine.execute("umount /sys/class/dmi")
 
+    def testCPUSecurityMitigations(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go('/system/hwinfo')
+        b.enter_page('/system/hwinfo')
```

Comment:
You don't need `enter_page()` after `login_and_go()`. That  does two more frame switches which are unnecessary (and just a potential flake).